docs/pages/federating-your-data/generating-tokens.mdx

```diff
@@ -0,0 +1,26 @@
+# Generating Tokens for Accessing Protected Objects
+
+To access a protected object in the Pelican data federation, the client must present a token signed by the Origin where the object is hosted on. The token then is used by Pelican servers to verify the object access permissions granted to the user, alone with other security checks to ensure the token is valid and not tempered by malicious attackers. Pelican takes advantage of JSON Web Token (JWT) to embed permission, user and server identity to the token.
+
+To allow user access the protected objects, the token must be generated by the Pelican Origin. Pelican Origin can be configured to use a third-party OIDC token issuer, in which case the client will be prompted to get a token from the issuer. By default, however, the Pelican Origin needs to generate the token by its built-in issuer and pass it to the client. This page documents different ways to generate the token to access the protected objects.
+
+## Pelican CLI
+
+The Pelican binary comes with a command `pelican origin token create` to generate the token. To generate a valid token, this command **MUST** be run on the same server where the Pelican Origin hosting the target object runs. On the Origin server, run:
+
+```bash copy
+pelican origin token create \
+    --scope "storage.read:/" \
+    --issuer https://localhost:8443 \
+    --claim "wlcg.ver=1.0" \
+    --subject origin \
+    --audience https://wlcg.cern.ch/jwt/v1/any
```

Comment:
Let's also document the `--private-key` flag, which lets you generate a token by pointing at a specific private key, and the `--lifetime` flag, which lets you set a token expiration value in seconds.